### PR TITLE
implement sending Point module config from UI

### LINKED ISSUE #3: Re-unite Point endpoint with its config
In the POC we use to be able to send config from the client to the host. This seems to have regressed because the relevant code looks commented out: https://github.com/moja-global/FLINT-UI/blob/14eb10f0a18791250dd2b028d7bd98e0fe7fa7ed/flint.ui/src/components/Cards/CardInfoRun.vue#L164

Please describe the steps needed to recreate this functionality.

### PR DESCRIPTION
**Pull request by shubhamkarande13**
# Implement sending Point module config from UI

## Description

This PR adds the functionality to send the Point module config from UI after setting custom values for Pools.

Fixes #3

### CODE REVIEW

File: flint.ui/src/store/index.js [hunk 1]
```diff
@@ -1,220 +1,409 @@
...
+      state.Point_config["Pools"][1]["Pool 2"] = pool_2_valuex;
+    },
+    set_pointConfig_Pool_3(state, pool_3_value) {
+      var pool_3_valuex = "#$" + pool_3_value + "#$";
+      state.Point_config["Pools"][2]["Pool 3"] = pool_3_valuex;
+      console.log(state.Point_config);
+    },
+  },
+  actions: {
+    send_pointConfig() {
```
Comment (aornugent) [inline flint.ui/src/store/index.js:392]: This is cool! Two thoughts:
1. Is store/index.js going to be the final home of all our API calls, or is there a better way to structure our application? @waridrox do you know if we can make this more modular and have e.g. store/point.js, store/rothc.js
2. Let's write a custom stringify function that can be used for all JSON dispatch, following the DRY principle.

File: flint.ui/src/store/index.js [hunk 2]
```diff
@@ -1,220 +1,409 @@
...
+    setNewConfig(state, newValue) {
+      state.Variables[8].rainfall.transform.data_orig = newValue;
     },
-    modules: {
-
-    }
-
-})
\ No newline at end of file
+    set_pointConfig_Pool_1(state, pool_1_value) {
```
Comment (aornugent) [inline flint.ui/src/store/index.js:376]: Can you generalise this by also passing in the Pool index?

Comment (shubhamkarande13) [reply flint.ui/src/store/index.js:376]: I did try that, but a single mutation makes only one change at a time. If we try to do multiple changes in the state, then we get the changes being passed as _'undefined'_.

image

File: flint.ui/src/store/index.js (hunk 1 above)
Comment (shubhamkarande13) [reply flint.ui/src/store/index.js:392]: @waridrox I guess you are already done with RothC in the store?

**Pull request merged**